ext/opentelemetry-exporter-cloud-trace/src/opentelemetry/exporter/cloud_trace/__init__.py

```diff
@@ -54,15 +54,23 @@
 from opentelemetry.sdk.trace import Event
 from opentelemetry.sdk.trace.export import Span, SpanExporter, SpanExportResult
 from opentelemetry.sdk.util import BoundedDict
+from opentelemetry.sdk.version import __version__ as core_version
 from opentelemetry.util import types
 
+# pylint: disable=import-error
+from .version import __version__ as cloud_trace_version
+
 logger = logging.getLogger(__name__)
 
 MAX_NUM_LINKS = 128
 MAX_NUM_EVENTS = 32
 MAX_EVENT_ATTRS = 4
 MAX_LINK_ATTRS = 32
 MAX_SPAN_ATTRS = 32
+AGENT_LABEL_KEY = "g.co/agent"
+AGENT_LABEL_VALUE = "opentelemetry-python {}; google-cloud-trace-exporter {}".format(
```

Comment:
Not `opentelemetry-exporter-cloud-trace`?

Interesting that black won't let me split this line...

Reply:
Dan recommended we follow the format here https://github.com/GoogleCloudPlatform/opentelemetry-operations-js/blob/d5c75f4e5ce35693c3c552b42004a5727dc36267/packages/opentelemetry-cloud-trace-exporter/src/transform.ts#L34.

Perhaps black is fine with global variables?